tcms/kiwi_auth/tests/test_admin.py

```diff
@@ -173,6 +173,9 @@ def test_should_be_allowed_to_create_new_group(self):
             'maxlength="150" required id="id_name">',
         )
 
+        # check for the user widget
+        self.assertContains(response, '<label for="id_users">%s</label>' % _("Users"))
```

Comment:
Probably using the wrong source string b/c testing in French fails. See the logs.

Reply:
Still failing.

Reply:
I searched django translations but only found 'msgid "users" ' , the capitalize will not work I guess.

Reply:
django/contrib/auth/forms.py - 202 & 208